_posts/blog/2018-09-05/2018-09-05-troubleshooting-and-how-to-help-with-development-on-openbazaar.markdown

```diff
@@ -0,0 +1,110 @@
+---
+title: "<span class='post-title-extra'>Troubleshooting</span> and How to Help with The <span class='post-title-extra'>Development</span> of OpenBazaar"
+layout: post
+date: '2018-09-05 08:30:00 -0600'
+social_title: 'Troubleshooting and How to Help with The Development of OpenBazaar'
+social_description: 'Troubleshooting and How to Help with The Development of OpenBazaar'
+---
+
+_By OpenBazaar Backend Developer MG_
+
+When we use any kind of software we are likely to encounter issues from time to time. With closed-source software, or software that we generally pay for, there's usually an easy way to contact someone for support as needed. You pay them, they fix your issues, it's transactional and generally very closed to be a one-on-one discussion.
+
+But there's a tradeoff. If that company changes their policies, pumps up their prices, or shuts down, these will have some effect on your ability to get that help--or even _keep using the software at all._ 
+
+[Open source software](https://coincenter.org/entry/what-is-open-source-and-why-is-it-important-for-cryptocurrency-and-open-blockchain-projects) is a different kind of animal. It's the result of many people coming together, sometimes from all around the world, to build programs that are transparent and inclusive, where contribution to the code base is easy and collaborative. This decentralization of technology is a major part of what we love about cryptocurrencies like [Bitcoin](https://bitcoin.org/) because **it's all about giving power back to individual users and contributors**. It allows them to be in control and make decisions for themselves instead leaving them up to large, centralized organizations.
+
+[OpenBazaar](https://openbazaar.org/download) is an open source application that is designed to be the fabric of future ecommerce where no middlemen or intermediaries are involved. And sometimes it presents some unique challenges. As the development leaders of this marketplace, we at [OB1](https://ob1.io) are focused on providing a quality user experience _within_ the app but also want to ensure users are empowered _outside_ of it as well so they can troubleshoot issues and help the platform keep growing. Which brings us to today’s topic...
+
+**Troubleshooting and Reporting Issues with OpenBazaar! (Cue applause)**
+
+Despite the careful attention given to developing the software, there will be bugs. And OpenBazaar not only has the benefit of a company like OB1 ensuring its team is available, transparent and engaged with their community when people need assistance, but it also has the communal benefits of an open source project. 
+
+Being able to navigate these resources efficiently could save you time and headaches but may be different than what you're accustomed to. This post should help clear up the support options that are available when you need help and show you how to give back to the OpenBazaar community at the same time.
```

Comment:
No resources have been mentioned yet, it's not clear what that refers to. Maybe "navigate the resources of an open source project efficiently".